### ISSUE TITLE
Gendustry combs...

### ISSUE THREAD

**Issue by Cinobi** [GTNewHorizons/NewHorizons#2249]
#### Which modpack version are you using?
Delta one
#
#### If in multiplayer; On which server does this happen? 
Delta
#
#### What did you try to do, and what did you expect to happen?
... can not be processed in Forestry centrifuges.(u can't even insert the combs in them). 
![2017-12-09_20 30 12](https://user-images.githubusercontent.com/25028786/33797837-e5d5761a-dd1f-11e7-9682-e9f949cd7c2d.png)

**Comment by Dream-Master** [comment]
Yes i disabled it and gated it in MV Tier. In Forestry Centrifuge  i can't gated recipes.
